vignettes/user.Rmd

```diff
@@ -0,0 +1,73 @@
+---
+title: "Scripts"
+output: rmarkdown::html_vignette
+vignette: >
+  %\VignetteIndexEntry{Scripts}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+```{r, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>"
+)
+options("lifecycle:::calling_package" = "tibble")
+```
+
+The lifecycle package uses warnings to advertise which functions are deprecated.
+Deprecated functions will be removed in a future release, so it's good practice to eliminate the warnings as soon as you see them.
+
+```{r setup}
+library(lifecycle)
+```
+
+For example, lets imagine your code used `tibble::data_frame()`, which was deprecated in favour of `tibble()` in version 1.1.0.
+`data_frame()` now looks something like this:
+
+```{r}
+data_frame <- function(...) {
+  deprecate_warn("1.1.0", "data_frame()", "tibble()")
+  tibble::tibble(...)
+}
+```
+
+So if you use `data_frame()` in your own code you'll get a warning:
+
+```{r}
```

Comment:
Hard-code the output in prevision of `data_frame()` becoming defunct?

Reply:
I defined my own `data_frame()` above 😄